weed/storage/needle_map_leveldb.go

```diff
@@ -17,10 +19,15 @@ import (
 	. "github.com/chrislusf/seaweedfs/weed/storage/types"
 )
 
+//mark it every milestoneBatchSize operations
+const milestoneBatchSize = 10000
+const milestoneKey = 0xffffffffffffffff - 1
```

Comment:
use a textual key. how about `[]byte("idx_entry_watermark")` ?

Please also change `milestone` to `watermark` throughout this PR. `watermark` I personally feel is easier to understand.

Reply:
done